packages/kbn-build/src/production/build_production_projects.integration_test.js

```diff
@@ -44,15 +37,6 @@ describe('kbn-build production', function() {
       cwd: buildRoot,
     });
 
-    expect(files).to.eql([
-      'packages/bar/package.json',
-      'packages/bar/src/index.js',
-      'packages/bar/target/index.js',
-      'packages/bar/yarn.lock',
-      'packages/foo/package.json',
-      'packages/foo/src/index.js',
-      'packages/foo/target/index.js',
-      'packages/foo/yarn.lock',
-    ]);
+    expect(files.sort()).toMatchSnapshot();
```

Comment:
__note:__ using `sort` here since `globby: 8.*` (that I'm planning to upgrade to in https://github.com/elastic/kibana/pull/16732) isn't consistent on the order of files for some reason and this test doesn't really care about the order as well.